src/community/label_propagation.jl

```diff
@@ -1,24 +1,24 @@
 """
 Community detection using the label propagation algorithm (see [Raghavan et al.](http://arxiv.org/abs/0709.2938)).
-`g`: imput Graph
+`g`: input Graph
 `maxiter`: maximum number of iterations
 return : vertex assignments and the convergence history
 """
```

Comment:
can you modify the docstring to 
"""
     label_propagation(g; maxiter=1000)

 Community detection using the label propagation algorithm (see [Raghavan et al.](http://arxiv.org/abs/0709.2938)).
 `maxiter` is the maximum number of iterations. Returns the vertex assignments and the convergence history
"""